src/pages/home/report/ReportActionCompose.js

```diff
@@ -687,7 +801,10 @@ class ReportActionCompose extends React.Component {
                                             isFullComposerAvailable={this.state.isFullComposerAvailable}
                                             setIsFullComposerAvailable={this.setIsFullComposerAvailable}
                                             isComposerFullSize={this.props.isComposerFullSize}
+                                            isSmallScreenWidth={this.props.isSmallScreenWidth}
```

Comment:
You should never have to drill this prop down since it comes from the `withWindowDimensions` HOC